src/components/ClusterManager.js

```diff
@@ -363,6 +388,23 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  async componentDidUpdate(prevProps) {
+    const { ajax: { Jupyter } } = this.props
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const { status, jupyterUserScriptUri, googleProject, clusterName, stagingBucket } = this.getCurrentCluster() || {}
+
+    if (status === 'Error' && prevCluster.status !== 'Error' && !errorNotified.current) {
+      const { errors: clusterErrors } = await Jupyter.cluster(googleProject, clusterName).details()
+      this.setState({ clusterErrors })
+      if (_.some({ errorMessage: 'Userscript failed.' }, clusterErrors)) {
```

Comment:
If the cluster errors for some reason unrelated to the user script, this won't show any notifications. Is that what we want?